### PR TITLE
docs: live explain components with Vocs

### PR DESCRIPTION
**Pull request by Zizzamia**
**What changed? Why?**
Ok, this is getting interesting.

Screenshot 2024-03-20 at 3 34 19 PM

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: site/tsconfig.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-    "skipLibCheck": true,
-    "moduleResolution": "bundler",
     "allowImportingTsExtensions": true,
-    "resolveJsonModule": true,
     "isolatedModules": true,
-    "noEmit": true,
     "jsx": "react-jsx",
-    "strict": true,
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
```
Comment (Zizzamia) [inline site/tsconfig.json:13]: By changing these to
```
    "module": "NodeNext",
    "moduleResolution": "NodeNext",
```
it did the trick.

cc @kyhyco @cnasc

File: site/tsconfig.json [hunk 2]
```diff
@@ -1,24 +1,24 @@
 {
   "compilerOptions": {
     "target": "ES2020",
-    "useDefineForClassFields": true,
```
Comment (Zizzamia) [inline site/tsconfig.json:4]: I did some sorting.

File: site/docs/pages/identity/App.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+'use client';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { ReactNode } from 'react';
+
+const queryClient = new QueryClient();
+
```
Comment (Zizzamia) [inline site/docs/pages/identity/App.tsx:6]: Btw very neat this way of testing.
thx @kyhyco @cnasc

File: site/tsconfig.json (hunk 1 above)
Comment (kyhyco) [reply site/tsconfig.json:13]: ahhh nice one!

**Pull request merged**